src/status_im2/common/contacts/style.cljs

```diff
@@ -0,0 +1,105 @@
+(ns status-im2.common.contacts.style
+  (:require [quo2.foundations.colors :as colors]
+            [quo2.foundations.typography :as typography]))
+
+(def content-outer-view
+  {:position         :absolute
+   :height           636
+   :top              132
+   :left             0
+   :right            0
+   :background-color colors/white
+   :padding          16
+   :border-radius    20})
+
+(def content-inner-view
+  {:flex           1
+   :flex-direction :column
+   :align-items    :flex-start
+   :position       :absolute
+   :width          375
+   :height         242})
+
+(def text-combination-view
+  {:flex-direction     :column
+   :align-items        :flex-start
+   :width              375
+   :height             98
+   :padding-top        16
+   :padding-bottom     20
+   :padding-horizontal 20
+   :justify-content    :space-between})
+
+(def key-view
+  {:flex           1
+   :flex-direction :row
+   :align-items    :flex-start
+   :width          375
+   :height         110
+   :padding-top    0
+   :padding-left   20
+   :padding-right  0
+   :padding-bottom 20})
+
+(def input-outer-view
+  {:flex           1
+   :flex-direction :column
+   :align-items    :flex-start
+   :margin-right   74})
+
+(def info-message-view
+  {:flex-direction     :row
+   :align-items        :flex-start
+   :padding-vertical   8
+   :padding-horizontal 20})
+
+(def bottom-actions-view
+  {:flex           1
+   :flex-direction :column
+   :align-items    :flex-start
+   :width          375
+   :height         98
+   :left           0
+   :top            538})
+
+(defn text-input
+  [state error]
+  (merge typography/monospace
+         typography/paragraph-1
+         {:padding-top      8
+          :padding-left     12
+          :padding-right    12
+          :padding-bottom   8
+          :align-self       :stretch
+          :background-color colors/white
+          :border-width     1
+          :border-radius    12
+          :border-color     (if (and (= state :error)
+                                     (= error :uncompressed-key))
```

Comment:
I think you can move this logic to the component and just use a flag in the style. For me it looks weird having this kind of logic in the styles file.

Reply:
yes, I agree. done